readme.md

```diff
@@ -32,9 +35,18 @@ pokemon.getId('Dratini');
 
 ## API
 
-### .all: *string[]*
+### .all(lang: *string*): *string[]*
+
+Get all names.
+
+With the optional `lang` parameter you can get a localized name using a [language code](https://en.wikipedia.org/wiki/ISO_639-1):
```

Comment:
I'd suggest formatting it like this 

``````
### .all(lang: *string* = 'en'): *string[]*

Localised list of all the names.

#### lang

[Language code](#supported-languages) to retrieve the list of pokémon for.

```js
pokemon.all();
//=> ['Bulbasaur', ...]

pokemon.all('de');
//=> ['Bisasam', ...]
```
``````

But I let @sindresorhus decide what he wants.